packages/cra-template/template/src/index.js

```diff
@@ -15,3 +16,8 @@ ReactDOM.render(
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://cra.link/PWA
 serviceWorker.unregister();
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```

Comment:
Would it be better to not include the function (and import) in the template and show how to add in the docs instead?

Reply:
@iansu also would it be possible to create a cra.link for the docs page?

Reply:
> Would it be better to not include the function (and import) in the template and show how to add in the docs instead?

If the unused code gets stripped out from the bundles it should be fine (but not sure if they do)!

Reply:
It does :))

Lib is dynamically imported only if a function gets passed in